web/platform/src/content/docs/docs/config/basic-configs.mdx

```diff
@@ -42,7 +39,7 @@ memory and filesystem stores instead of S3 and Redis.
             "temp_path": "/tmp/nativelink/data-worker-test/tmp_path-cas",
             "eviction_policy": {
               // 10gb.
```

Comment:
Comment can go away now it's written out

Reply:
agree